server/src/test/java/org/elasticsearch/search/aggregations/bucket/geogrid/GeoGridAggregatorTestCase.java

```diff
@@ -112,17 +122,33 @@ public void testWithSeveralDocs() throws IOException {
             if (points.size() != 0) {
                 iw.addDocument(points);
             }
-        }, geoHashGrid -> {
-            assertEquals(expectedCountPerGeoHash.size(), geoHashGrid.getBuckets().size());
-            for (GeoGrid.Bucket bucket : geoHashGrid.getBuckets()) {
-                assertEquals((long) expectedCountPerGeoHash.get(bucket.getKeyAsString()), bucket.getDocCount());
-            }
-            assertTrue(AggregationInspectionHelper.hasValue(geoHashGrid));
         });
     }
 
-    private void testCase(Query query, String field, int precision, CheckedConsumer<RandomIndexWriter, IOException> buildIndex,
-                          Consumer<InternalGeoGrid<T>> verify) throws IOException {
+    public void testBounds() throws IOException {
```

Comment:
Can we randomize the test? e.g., `GeometryTestUtils.randomRectangle` and add doc value points somewhere near the middle of the random generated rectangle?

Reply:
for sure. I'll randomize. I felt like I could by-pass randomization here due to 
the underlying `pointInBounds` already having randomized tests in `testPointInBounds`